static/js/HitchhikerJS.js

```diff
@@ -35,8 +35,15 @@ import RuntimeConfigReceiver from './runtime-config-receiver';
 export { RuntimeConfigReceiver };
 
 import RuntimeConfig from './runtime-config';
-import AnswersExperience from './answers-experience';
 const runtimeConfig = new RuntimeConfig();
+
+import AnswersExperience from './answers-experience';
 window.AnswersExperience = new AnswersExperience(runtimeConfig);
 
+import analyticsListener from './runtime-config-listeners/analytics';
```

Comment:
would it make more sense to do the registerListener calls inside AnswersExperience?

Reply:
pipe dream but maybe there's a world where we manage to move all the cards into the bundle, and then move the ANSWERS.init into AnswersExperience?

Reply:
ya the, listener registration does make sense to do in the AnswersExperience

Reply:
Moving our JS out of html.hbs and core.hbs into HitchhikerJS would be great

Reply:
updated to move registerListener calls to AnswersExperience